bpf/bpf_lxc.c

```diff
@@ -86,7 +86,7 @@ static __always_inline int ipv6_l3_from_lxc(struct __ctx_buff *ctx,
 					    int l3_off, struct ipv6hdr *ip6,
 					    __u32 *dstID)
 {
-#ifdef ENABLE_ROUTING
+#ifndef ENABLE_ENDPOINT_ROUTES
```

Comment:
Can we please remove this one and just change the declaration below to tell the compiler that the variable may be unused? Then it should automatically be compiled out if the variable is not needed. Like this:
```c
	union macaddr __maybe_unused router_mac = NODE_MAC;
```

The fewer `#ifdef` we have, the more readable the code will be.

Reply:
@qmonnet thank you for suggestion and it was addressed now.